docs/ref/contrib/admin/index.txt

```diff
@@ -1115,7 +1115,9 @@ subclass::
     more user-friendly if the related model has many instances.
 
     You must define :attr:`~ModelAdmin.search_fields` on the related object's
-    ``ModelAdmin`` because the autocomplete search uses it.
+    ``ModelAdmin`` because the autocomplete search uses it. (In addtion, this
```

Comment:
I would drop the parentheses, "this means that", and "in order to use autocomplete".

Reply:
addtion -> addition